src/renault_api/kamereon/models.py

```diff
@@ -1,20 +1,41 @@
 """Kamereon models."""
 import json
 from dataclasses import dataclass
-from enum import Enum
 from typing import Any
 from typing import cast
 from typing import Dict
 from typing import List
 from typing import Optional
 
-import marshmallow_dataclass
 from marshmallow.schema import Schema
 
-from . import BaseModel
-from . import BaseSchema
-from renault_api.exceptions import KamereonException
-from renault_api.exceptions import KamereonResponseException
+from . import enums
+from . import exceptions
+from renault_api.models import BaseModel
+
+COMMON_ERRRORS: List[Dict[str, Any]] = [
+    {
+        "errorCode": "err.func.403",
+        "error_details": "Access is denied for this resource",
+        "error_type": exceptions.AccessDeniedException,
+    },
+    {
+        "errorCode": "err.tech.500",
+        "error_details": "Invalid response from the upstream server"
+        " (The request sent to the GDC is erroneous) ; 502 Bad Gateway",
+        "error_type": exceptions.InvalidUpstreamException,
+    },
+    {
+        "errorCode": "err.tech.501",
+        "error_details": "This feature is not technically supported by this gateway",
+        "error_type": exceptions.NotSupportedException,
+    },
+    {
+        "errorCode": "err.func.wired.overloaded",
+        "error_details": "You have reached your quota limit",
+        "error_type": exceptions.QuotaLimitException,
+    },
+]
```

Comment:
```suggestion
COMMON_ERRRORS: List[Dict[str, Any]] = [
    {
        "errorCode": "err.func.403",
        "error_type": exceptions.AccessDeniedException,
    },
    {
        "errorCode": "err.tech.500",
        "error_type": exceptions.InvalidUpstreamException,
    },
    {
        "errorCode": "err.tech.501",
        "error_type": exceptions.NotSupportedException,
    },
    {
        "errorCode": "err.func.wired.overloaded",
        "error_type": exceptions.QuotaLimitException,
    },
]
```

As previous comment

Reply:
Addressed in 1e48d0038b3e2bece4ed476e080271f1caa0d36f